addons/csi/vsphere/snapshot-controller.yaml

```diff
@@ -0,0 +1,154 @@
+# Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sourced from:
+# - https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.1/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
+# - https://github.com/kubernetes-csi/external-snapshotter/blob/release-4.1/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml
+# Modifications:
+#   - Set image tag to 5.0.1 to match the release version
```

Comment:
This doesn't seem to be accurate, the image tag is `v4.0.0` in this file?

Reply:
Yes, I templated the version according to the csi driver version